/home/gcasadella/miniconda3/envs/openmmlab/lib/python3.8/site-packages/torch/distributed/launch.py:183: FutureWarning: The module torch.distributed.launch is deprecated
and will be removed in future. Use torchrun.
Note that --use-env is set by default in torchrun.
If your script expects `--local-rank` argument to be set, please
change it to read from `os.environ['LOCAL_RANK']` instead. See 
https://pytorch.org/docs/stable/distributed.html#launch-utility for 
further instructions

  warnings.warn(
[2024-03-07 18:42:58,571] torch.distributed.run: [WARNING] 
[2024-03-07 18:42:58,571] torch.distributed.run: [WARNING] *****************************************
[2024-03-07 18:42:58,571] torch.distributed.run: [WARNING] Setting OMP_NUM_THREADS environment variable for each process to be 1 in default, to avoid your system being overloaded, please further tune the variable for optimal performance in your application as needed. 
[2024-03-07 18:42:58,571] torch.distributed.run: [WARNING] *****************************************
[W socket.cpp:464] [c10d] The server socket has failed to bind to [::]:29500 (errno: 98 - Address already in use).
[W socket.cpp:464] [c10d] The server socket has failed to bind to 0.0.0.0:29500 (errno: 98 - Address already in use).
[E socket.cpp:500] [c10d] The server socket has failed to listen on any local network address.
Traceback (most recent call last):
  File "/home/gcasadella/miniconda3/envs/openmmlab/lib/python3.8/runpy.py", line 194, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "/home/gcasadella/miniconda3/envs/openmmlab/lib/python3.8/runpy.py", line 87, in _run_code
    exec(code, run_globals)
  File "/home/gcasadella/miniconda3/envs/openmmlab/lib/python3.8/site-packages/torch/distributed/launch.py", line 198, in <module>
    main()
  File "/home/gcasadella/miniconda3/envs/openmmlab/lib/python3.8/site-packages/torch/distributed/launch.py", line 194, in main
    launch(args)
  File "/home/gcasadella/miniconda3/envs/openmmlab/lib/python3.8/site-packages/torch/distributed/launch.py", line 179, in launch
    run(args)
  File "/home/gcasadella/miniconda3/envs/openmmlab/lib/python3.8/site-packages/torch/distributed/run.py", line 803, in run
    elastic_launch(
  File "/home/gcasadella/miniconda3/envs/openmmlab/lib/python3.8/site-packages/torch/distributed/launcher/api.py", line 135, in __call__
    return launch_agent(self._config, self._entrypoint, list(args))
  File "/home/gcasadella/miniconda3/envs/openmmlab/lib/python3.8/site-packages/torch/distributed/launcher/api.py", line 259, in launch_agent
    result = agent.run()
  File "/home/gcasadella/miniconda3/envs/openmmlab/lib/python3.8/site-packages/torch/distributed/elastic/metrics/api.py", line 123, in wrapper
    result = f(*args, **kwargs)
  File "/home/gcasadella/miniconda3/envs/openmmlab/lib/python3.8/site-packages/torch/distributed/elastic/agent/server/api.py", line 727, in run
    result = self._invoke_run(role)
  File "/home/gcasadella/miniconda3/envs/openmmlab/lib/python3.8/site-packages/torch/distributed/elastic/agent/server/api.py", line 862, in _invoke_run
    self._initialize_workers(self._worker_group)
  File "/home/gcasadella/miniconda3/envs/openmmlab/lib/python3.8/site-packages/torch/distributed/elastic/metrics/api.py", line 123, in wrapper
    result = f(*args, **kwargs)
  File "/home/gcasadella/miniconda3/envs/openmmlab/lib/python3.8/site-packages/torch/distributed/elastic/agent/server/api.py", line 699, in _initialize_workers
    self._rendezvous(worker_group)
  File "/home/gcasadella/miniconda3/envs/openmmlab/lib/python3.8/site-packages/torch/distributed/elastic/metrics/api.py", line 123, in wrapper
    result = f(*args, **kwargs)
  File "/home/gcasadella/miniconda3/envs/openmmlab/lib/python3.8/site-packages/torch/distributed/elastic/agent/server/api.py", line 542, in _rendezvous
    store, group_rank, group_world_size = spec.rdzv_handler.next_rendezvous()
  File "/home/gcasadella/miniconda3/envs/openmmlab/lib/python3.8/site-packages/torch/distributed/elastic/rendezvous/static_tcp_rendezvous.py", line 55, in next_rendezvous
    self._store = TCPStore(  # type: ignore[call-arg]
torch.distributed.DistNetworkError: The server socket has failed to listen on any local network address. The server socket has failed to bind to [::]:29500 (errno: 98 - Address already in use). The server socket has failed to bind to 0.0.0.0:29500 (errno: 98 - Address already in use).
2024-03-07 19:32:02.312872: I tensorflow/tsl/cuda/cudart_stub.cc:28] Could not find cuda drivers on your machine, GPU will not be used.
2024-03-07 19:32:02.364702: I tensorflow/tsl/cuda/cudart_stub.cc:28] Could not find cuda drivers on your machine, GPU will not be used.
2024-03-07 19:32:02.365258: I tensorflow/core/platform/cpu_feature_guard.cc:182] This TensorFlow binary is optimized to use available CPU instructions in performance-critical operations.
To enable the following instructions: AVX2 FMA, in other operations, rebuild TensorFlow with the appropriate compiler flags.
2024-03-07 19:32:03.104610: W tensorflow/compiler/tf2tensorrt/utils/py_utils.cc:38] TF-TRT Warning: Could not find TensorRT
2024-03-07 19:32:04.332979: W tensorflow/core/common_runtime/gpu/gpu_device.cc:1960] Cannot dlopen some GPU libraries. Please make sure the missing libraries mentioned above are installed properly if you would like to use GPU. Follow the guide at https://www.tensorflow.org/install/gpu for how to download and setup the required libraries for your platform.
Skipping registering GPU devices...

NOTE: Using experimental fast data loading logic. To disable, pass
    "--load_fast=false" and report issues on GitHub. More details:
    https://github.com/tensorflow/tensorboard/issues/4784

TensorBoard 2.13.0 at http://cudahpc09:6006/ (Press CTRL+C to quit)
2024-03-08 15:48:36.182525: I tensorflow/tsl/cuda/cudart_stub.cc:28] Could not find cuda drivers on your machine, GPU will not be used.
2024-03-08 15:48:36.227347: I tensorflow/tsl/cuda/cudart_stub.cc:28] Could not find cuda drivers on your machine, GPU will not be used.
2024-03-08 15:48:36.227902: I tensorflow/core/platform/cpu_feature_guard.cc:182] This TensorFlow binary is optimized to use available CPU instructions in performance-critical operations.
To enable the following instructions: AVX2 FMA, in other operations, rebuild TensorFlow with the appropriate compiler flags.
2024-03-08 15:48:36.940609: W tensorflow/compiler/tf2tensorrt/utils/py_utils.cc:38] TF-TRT Warning: Could not find TensorRT
2024-03-08 15:48:37.983968: W tensorflow/core/common_runtime/gpu/gpu_device.cc:1960] Cannot dlopen some GPU libraries. Please make sure the missing libraries mentioned above are installed properly if you would like to use GPU. Follow the guide at https://www.tensorflow.org/install/gpu for how to download and setup the required libraries for your platform.
Skipping registering GPU devices...

NOTE: Using experimental fast data loading logic. To disable, pass
    "--load_fast=false" and report issues on GitHub. More details:
    https://github.com/tensorflow/tensorboard/issues/4784

TensorBoard 2.13.0 at http://cudahpc09:6007/ (Press CTRL+C to quit)
2024-03-08 17:10:34.453496: I tensorflow/tsl/cuda/cudart_stub.cc:28] Could not find cuda drivers on your machine, GPU will not be used.
2024-03-08 17:10:34.509638: I tensorflow/tsl/cuda/cudart_stub.cc:28] Could not find cuda drivers on your machine, GPU will not be used.
2024-03-08 17:10:34.510252: I tensorflow/core/platform/cpu_feature_guard.cc:182] This TensorFlow binary is optimized to use available CPU instructions in performance-critical operations.
To enable the following instructions: AVX2 FMA, in other operations, rebuild TensorFlow with the appropriate compiler flags.
2024-03-08 17:10:35.331686: W tensorflow/compiler/tf2tensorrt/utils/py_utils.cc:38] TF-TRT Warning: Could not find TensorRT
2024-03-08 17:10:36.535103: W tensorflow/core/common_runtime/gpu/gpu_device.cc:1960] Cannot dlopen some GPU libraries. Please make sure the missing libraries mentioned above are installed properly if you would like to use GPU. Follow the guide at https://www.tensorflow.org/install/gpu for how to download and setup the required libraries for your platform.
Skipping registering GPU devices...

NOTE: Using experimental fast data loading logic. To disable, pass
    "--load_fast=false" and report issues on GitHub. More details:
    https://github.com/tensorflow/tensorboard/issues/4784

TensorBoard 2.13.0 at http://cudahpc09:6008/ (Press CTRL+C to quit)
TensorBoard caught SIGTERM; exiting...
TensorBoard caught SIGTERM; exiting...
TensorBoard caught SIGTERM; exiting...
2024-03-08 19:10:08.996141: I tensorflow/tsl/cuda/cudart_stub.cc:28] Could not find cuda drivers on your machine, GPU will not be used.
2024-03-08 19:10:09.063980: I tensorflow/tsl/cuda/cudart_stub.cc:28] Could not find cuda drivers on your machine, GPU will not be used.
2024-03-08 19:10:09.064660: I tensorflow/core/platform/cpu_feature_guard.cc:182] This TensorFlow binary is optimized to use available CPU instructions in performance-critical operations.
To enable the following instructions: AVX2 FMA, in other operations, rebuild TensorFlow with the appropriate compiler flags.
2024-03-08 19:10:10.013338: W tensorflow/compiler/tf2tensorrt/utils/py_utils.cc:38] TF-TRT Warning: Could not find TensorRT
2024-03-08 19:10:11.577775: W tensorflow/core/common_runtime/gpu/gpu_device.cc:1960] Cannot dlopen some GPU libraries. Please make sure the missing libraries mentioned above are installed properly if you would like to use GPU. Follow the guide at https://www.tensorflow.org/install/gpu for how to download and setup the required libraries for your platform.
Skipping registering GPU devices...

NOTE: Using experimental fast data loading logic. To disable, pass
    "--load_fast=false" and report issues on GitHub. More details:
    https://github.com/tensorflow/tensorboard/issues/4784

TensorBoard 2.13.0 at http://cudahpc09:6000/ (Press CTRL+C to quit)
2024-03-08 19:10:37.683889: I tensorflow/tsl/cuda/cudart_stub.cc:28] Could not find cuda drivers on your machine, GPU will not be used.
2024-03-08 19:10:37.739093: I tensorflow/tsl/cuda/cudart_stub.cc:28] Could not find cuda drivers on your machine, GPU will not be used.
2024-03-08 19:10:37.739772: I tensorflow/core/platform/cpu_feature_guard.cc:182] This TensorFlow binary is optimized to use available CPU instructions in performance-critical operations.
To enable the following instructions: AVX2 FMA, in other operations, rebuild TensorFlow with the appropriate compiler flags.
2024-03-08 19:10:38.706439: W tensorflow/compiler/tf2tensorrt/utils/py_utils.cc:38] TF-TRT Warning: Could not find TensorRT
2024-03-08 19:10:40.158889: W tensorflow/core/common_runtime/gpu/gpu_device.cc:1960] Cannot dlopen some GPU libraries. Please make sure the missing libraries mentioned above are installed properly if you would like to use GPU. Follow the guide at https://www.tensorflow.org/install/gpu for how to download and setup the required libraries for your platform.
Skipping registering GPU devices...

NOTE: Using experimental fast data loading logic. To disable, pass
    "--load_fast=false" and report issues on GitHub. More details:
    https://github.com/tensorflow/tensorboard/issues/4784

Address already in use
Port 6000 is in use by another program. Either identify and stop that program, or start the server with a different port.
2024-03-08 19:10:58.664038: I tensorflow/tsl/cuda/cudart_stub.cc:28] Could not find cuda drivers on your machine, GPU will not be used.
2024-03-08 19:10:58.718726: I tensorflow/tsl/cuda/cudart_stub.cc:28] Could not find cuda drivers on your machine, GPU will not be used.
2024-03-08 19:10:58.719374: I tensorflow/core/platform/cpu_feature_guard.cc:182] This TensorFlow binary is optimized to use available CPU instructions in performance-critical operations.
To enable the following instructions: AVX2 FMA, in other operations, rebuild TensorFlow with the appropriate compiler flags.
2024-03-08 19:10:59.573887: W tensorflow/compiler/tf2tensorrt/utils/py_utils.cc:38] TF-TRT Warning: Could not find TensorRT
2024-03-08 19:11:01.210499: W tensorflow/core/common_runtime/gpu/gpu_device.cc:1960] Cannot dlopen some GPU libraries. Please make sure the missing libraries mentioned above are installed properly if you would like to use GPU. Follow the guide at https://www.tensorflow.org/install/gpu for how to download and setup the required libraries for your platform.
Skipping registering GPU devices...

NOTE: Using experimental fast data loading logic. To disable, pass
    "--load_fast=false" and report issues on GitHub. More details:
    https://github.com/tensorflow/tensorboard/issues/4784

TensorBoard 2.13.0 at http://cudahpc09:6006/ (Press CTRL+C to quit)
TensorBoard caught SIGTERM; exiting...
E0308 19:30:01.356951 139758126679616 _internal.py:96] Error on request:
Traceback (most recent call last):
  File "/home/gcasadella/miniconda3/envs/openmmlab/lib/python3.8/site-packages/werkzeug/serving.py", line 362, in run_wsgi
    execute(self.server.app)
  File "/home/gcasadella/miniconda3/envs/openmmlab/lib/python3.8/site-packages/werkzeug/serving.py", line 323, in execute
    application_iter = app(environ, start_response)
  File "/home/gcasadella/miniconda3/envs/openmmlab/lib/python3.8/site-packages/tensorboard/backend/application.py", line 528, in __call__
    return self._app(environ, start_response)
  File "/home/gcasadella/miniconda3/envs/openmmlab/lib/python3.8/site-packages/tensorboard/backend/application.py", line 569, in wrapper
    return wsgi_app(environ, start_response)
  File "/home/gcasadella/miniconda3/envs/openmmlab/lib/python3.8/site-packages/tensorboard/backend/security_validator.py", line 91, in __call__
    return self._application(environ, start_response_proxy)
  File "/home/gcasadella/miniconda3/envs/openmmlab/lib/python3.8/site-packages/tensorboard/backend/path_prefix.py", line 68, in __call__
    return self._application(environ, start_response)
  File "/home/gcasadella/miniconda3/envs/openmmlab/lib/python3.8/site-packages/tensorboard/backend/experiment_id.py", line 73, in __call__
    return self._application(environ, start_response)
  File "/home/gcasadella/miniconda3/envs/openmmlab/lib/python3.8/site-packages/tensorboard/backend/empty_path_redirect.py", line 43, in __call__
    return self._application(environ, start_response)
  File "/home/gcasadella/miniconda3/envs/openmmlab/lib/python3.8/site-packages/tensorboard/backend/client_feature_flags.py", line 55, in __call__
    return self._application(environ, start_response)
  File "/home/gcasadella/miniconda3/envs/openmmlab/lib/python3.8/site-packages/tensorboard/backend/auth_context_middleware.py", line 38, in __call__
    return self._application(environ, start_response)
  File "/home/gcasadella/miniconda3/envs/openmmlab/lib/python3.8/site-packages/tensorboard/backend/application.py", line 551, in _route_request
    return self.exact_routes[clean_path](environ, start_response)
  File "/home/gcasadella/miniconda3/envs/openmmlab/lib/python3.8/site-packages/werkzeug/wrappers/request.py", line 189, in application
    resp = f(*args[:-2] + (request,))
  File "/home/gcasadella/miniconda3/envs/openmmlab/lib/python3.8/site-packages/tensorboard/plugins/core/core_plugin.py", line 257, in _serve_runs
    self._data_provider.list_runs(ctx, experiment_id=experiment),
  File "/home/gcasadella/miniconda3/envs/openmmlab/lib/python3.8/site-packages/tensorboard/data/grpc_provider.py", line 76, in list_runs
    res = self._stub.ListRuns(req)
  File "/home/gcasadella/miniconda3/envs/openmmlab/lib/python3.8/site-packages/grpc/_channel.py", line 1160, in __call__
    return _end_unary_response_blocking(state, call, False, None)
  File "/home/gcasadella/miniconda3/envs/openmmlab/lib/python3.8/site-packages/grpc/_channel.py", line 1003, in _end_unary_response_blocking
    raise _InactiveRpcError(state)  # pytype: disable=not-instantiable
grpc._channel._InactiveRpcError: <_InactiveRpcError of RPC that terminated with:
	status = StatusCode.UNAVAILABLE
	details = "failed to connect to all addresses; last error: UNKNOWN: ipv4:127.0.0.1:36157: Failed to connect to remote host: Connection refused"
	debug_error_string = "UNKNOWN:Error received from peer  {grpc_message:"failed to connect to all addresses; last error: UNKNOWN: ipv4:127.0.0.1:36157: Failed to connect to remote host: Connection refused", grpc_status:14, created_time:"2024-03-08T19:30:01.332095414+01:00"}"
>
E0308 19:30:01.357749 139758042785344 _internal.py:96] Error on request:
Traceback (most recent call last):
  File "/home/gcasadella/miniconda3/envs/openmmlab/lib/python3.8/site-packages/werkzeug/serving.py", line 362, in run_wsgi
    execute(self.server.app)
  File "/home/gcasadella/miniconda3/envs/openmmlab/lib/python3.8/site-packages/werkzeug/serving.py", line 323, in execute
    application_iter = app(environ, start_response)
  File "/home/gcasadella/miniconda3/envs/openmmlab/lib/python3.8/site-packages/tensorboard/backend/application.py", line 528, in __call__
    return self._app(environ, start_response)
  File "/home/gcasadella/miniconda3/envs/openmmlab/lib/python3.8/site-packages/tensorboard/backend/application.py", line 569, in wrapper
    return wsgi_app(environ, start_response)
  File "/home/gcasadella/miniconda3/envs/openmmlab/lib/python3.8/site-packages/tensorboard/backend/security_validator.py", line 91, in __call__
    return self._application(environ, start_response_proxy)
  File "/home/gcasadella/miniconda3/envs/openmmlab/lib/python3.8/site-packages/tensorboard/backend/path_prefix.py", line 68, in __call__
    return self._application(environ, start_response)
  File "/home/gcasadella/miniconda3/envs/openmmlab/lib/python3.8/site-packages/tensorboard/backend/experiment_id.py", line 73, in __call__
    return self._application(environ, start_response)
  File "/home/gcasadella/miniconda3/envs/openmmlab/lib/python3.8/site-packages/tensorboard/backend/empty_path_redirect.py", line 43, in __call__
    return self._application(environ, start_response)
  File "/home/gcasadella/miniconda3/envs/openmmlab/lib/python3.8/site-packages/tensorboard/backend/client_feature_flags.py", line 55, in __call__
    return self._application(environ, start_response)
  File "/home/gcasadella/miniconda3/envs/openmmlab/lib/python3.8/site-packages/tensorboard/backend/auth_context_middleware.py", line 38, in __call__
    return self._application(environ, start_response)
  File "/home/gcasadella/miniconda3/envs/openmmlab/lib/python3.8/site-packages/tensorboard/backend/application.py", line 551, in _route_request
    return self.exact_routes[clean_path](environ, start_response)
  File "/home/gcasadella/miniconda3/envs/openmmlab/lib/python3.8/site-packages/werkzeug/wrappers/request.py", line 189, in application
    resp = f(*args[:-2] + (request,))
  File "/home/gcasadella/miniconda3/envs/openmmlab/lib/python3.8/site-packages/tensorboard/plugins/core/core_plugin.py", line 257, in _serve_runs
    self._data_provider.list_runs(ctx, experiment_id=experiment),
  File "/home/gcasadella/miniconda3/envs/openmmlab/lib/python3.8/site-packages/tensorboard/data/grpc_provider.py", line 76, in list_runs
    res = self._stub.ListRuns(req)
  File "/home/gcasadella/miniconda3/envs/openmmlab/lib/python3.8/site-packages/grpc/_channel.py", line 1160, in __call__
    return _end_unary_response_blocking(state, call, False, None)
  File "/home/gcasadella/miniconda3/envs/openmmlab/lib/python3.8/site-packages/grpc/_channel.py", line 1003, in _end_unary_response_blocking
    raise _InactiveRpcError(state)  # pytype: disable=not-instantiable
grpc._channel._InactiveRpcError: <_InactiveRpcError of RPC that terminated with:
	status = StatusCode.UNAVAILABLE
	details = "failed to connect to all addresses; last error: UNKNOWN: ipv4:127.0.0.1:36157: Failed to connect to remote host: Connection refused"
	debug_error_string = "UNKNOWN:Error received from peer  {created_time:"2024-03-08T19:30:01.34183672+01:00", grpc_status:14, grpc_message:"failed to connect to all addresses; last error: UNKNOWN: ipv4:127.0.0.1:36157: Failed to connect to remote host: Connection refused"}"
>
E0308 19:30:01.358079 139758084732480 _internal.py:96] Error on request:
Traceback (most recent call last):
  File "/home/gcasadella/miniconda3/envs/openmmlab/lib/python3.8/site-packages/werkzeug/serving.py", line 362, in run_wsgi
    execute(self.server.app)
  File "/home/gcasadella/miniconda3/envs/openmmlab/lib/python3.8/site-packages/werkzeug/serving.py", line 323, in execute
    application_iter = app(environ, start_response)
  File "/home/gcasadella/miniconda3/envs/openmmlab/lib/python3.8/site-packages/tensorboard/backend/application.py", line 528, in __call__
    return self._app(environ, start_response)
  File "/home/gcasadella/miniconda3/envs/openmmlab/lib/python3.8/site-packages/tensorboard/backend/application.py", line 569, in wrapper
    return wsgi_app(environ, start_response)
  File "/home/gcasadella/miniconda3/envs/openmmlab/lib/python3.8/site-packages/tensorboard/backend/security_validator.py", line 91, in __call__
    return self._application(environ, start_response_proxy)
  File "/home/gcasadella/miniconda3/envs/openmmlab/lib/python3.8/site-packages/tensorboard/backend/path_prefix.py", line 68, in __call__
    return self._application(environ, start_response)
  File "/home/gcasadella/miniconda3/envs/openmmlab/lib/python3.8/site-packages/tensorboard/backend/experiment_id.py", line 73, in __call__
    return self._application(environ, start_response)
  File "/home/gcasadella/miniconda3/envs/openmmlab/lib/python3.8/site-packages/tensorboard/backend/empty_path_redirect.py", line 43, in __call__
    return self._application(environ, start_response)
  File "/home/gcasadella/miniconda3/envs/openmmlab/lib/python3.8/site-packages/tensorboard/backend/client_feature_flags.py", line 55, in __call__
    return self._application(environ, start_response)
  File "/home/gcasadella/miniconda3/envs/openmmlab/lib/python3.8/site-packages/tensorboard/backend/auth_context_middleware.py", line 38, in __call__
    return self._application(environ, start_response)
  File "/home/gcasadella/miniconda3/envs/openmmlab/lib/python3.8/site-packages/tensorboard/backend/application.py", line 551, in _route_request
    return self.exact_routes[clean_path](environ, start_response)
  File "/home/gcasadella/miniconda3/envs/openmmlab/lib/python3.8/site-packages/werkzeug/wrappers/request.py", line 189, in application
    resp = f(*args[:-2] + (request,))
  File "/home/gcasadella/miniconda3/envs/openmmlab/lib/python3.8/site-packages/tensorboard/backend/application.py", line 412, in _serve_plugins_listing
    self._data_provider.list_plugins(ctx, experiment_id=eid)
  File "/home/gcasadella/miniconda3/envs/openmmlab/lib/python3.8/site-packages/tensorboard/data/grpc_provider.py", line 69, in list_plugins
    res = self._stub.ListPlugins(req)
  File "/home/gcasadella/miniconda3/envs/openmmlab/lib/python3.8/site-packages/grpc/_channel.py", line 1160, in __call__
    return _end_unary_response_blocking(state, call, False, None)
  File "/home/gcasadella/miniconda3/envs/openmmlab/lib/python3.8/site-packages/grpc/_channel.py", line 1003, in _end_unary_response_blocking
    raise _InactiveRpcError(state)  # pytype: disable=not-instantiable
grpc._channel._InactiveRpcError: <_InactiveRpcError of RPC that terminated with:
	status = StatusCode.UNAVAILABLE
	details = "failed to connect to all addresses; last error: UNKNOWN: ipv4:127.0.0.1:36157: Failed to connect to remote host: Connection refused"
	debug_error_string = "UNKNOWN:Error received from peer  {created_time:"2024-03-08T19:30:01.330357766+01:00", grpc_status:14, grpc_message:"failed to connect to all addresses; last error: UNKNOWN: ipv4:127.0.0.1:36157: Failed to connect to remote host: Connection refused"}"
>
E0308 19:30:01.358865 139758000838208 _internal.py:96] Error on request:
Traceback (most recent call last):
  File "/home/gcasadella/miniconda3/envs/openmmlab/lib/python3.8/site-packages/werkzeug/serving.py", line 362, in run_wsgi
    execute(self.server.app)
  File "/home/gcasadella/miniconda3/envs/openmmlab/lib/python3.8/site-packages/werkzeug/serving.py", line 323, in execute
    application_iter = app(environ, start_response)
  File "/home/gcasadella/miniconda3/envs/openmmlab/lib/python3.8/site-packages/tensorboard/backend/application.py", line 528, in __call__
    return self._app(environ, start_response)
  File "/home/gcasadella/miniconda3/envs/openmmlab/lib/python3.8/site-packages/tensorboard/backend/application.py", line 569, in wrapper
    return wsgi_app(environ, start_response)
  File "/home/gcasadella/miniconda3/envs/openmmlab/lib/python3.8/site-packages/tensorboard/backend/security_validator.py", line 91, in __call__
    return self._application(environ, start_response_proxy)
  File "/home/gcasadella/miniconda3/envs/openmmlab/lib/python3.8/site-packages/tensorboard/backend/path_prefix.py", line 68, in __call__
    return self._application(environ, start_response)
  File "/home/gcasadella/miniconda3/envs/openmmlab/lib/python3.8/site-packages/tensorboard/backend/experiment_id.py", line 73, in __call__
    return self._application(environ, start_response)
  File "/home/gcasadella/miniconda3/envs/openmmlab/lib/python3.8/site-packages/tensorboard/backend/empty_path_redirect.py", line 43, in __call__
    return self._application(environ, start_response)
  File "/home/gcasadella/miniconda3/envs/openmmlab/lib/python3.8/site-packages/tensorboard/backend/client_feature_flags.py", line 55, in __call__
    return self._application(environ, start_response)
  File "/home/gcasadella/miniconda3/envs/openmmlab/lib/python3.8/site-packages/tensorboard/backend/auth_context_middleware.py", line 38, in __call__
    return self._application(environ, start_response)
  File "/home/gcasadella/miniconda3/envs/openmmlab/lib/python3.8/site-packages/tensorboard/backend/application.py", line 551, in _route_request
    return self.exact_routes[clean_path](environ, start_response)
  File "/home/gcasadella/miniconda3/envs/openmmlab/lib/python3.8/site-packages/werkzeug/wrappers/request.py", line 189, in application
    resp = f(*args[:-2] + (request,))
  File "/home/gcasadella/miniconda3/envs/openmmlab/lib/python3.8/site-packages/tensorboard/plugins/core/core_plugin.py", line 183, in _serve_environment
    md = self._data_provider.experiment_metadata(
  File "/home/gcasadella/miniconda3/envs/openmmlab/lib/python3.8/site-packages/tensorboard/data/grpc_provider.py", line 56, in experiment_metadata
    res = self._stub.GetExperiment(req)
  File "/home/gcasadella/miniconda3/envs/openmmlab/lib/python3.8/site-packages/grpc/_channel.py", line 1160, in __call__
    return _end_unary_response_blocking(state, call, False, None)
  File "/home/gcasadella/miniconda3/envs/openmmlab/lib/python3.8/site-packages/grpc/_channel.py", line 1003, in _end_unary_response_blocking
    raise _InactiveRpcError(state)  # pytype: disable=not-instantiable
grpc._channel._InactiveRpcError: <_InactiveRpcError of RPC that terminated with:
	status = StatusCode.UNAVAILABLE
	details = "failed to connect to all addresses; last error: UNKNOWN: ipv4:127.0.0.1:36157: Failed to connect to remote host: Connection refused"
	debug_error_string = "UNKNOWN:Error received from peer  {grpc_message:"failed to connect to all addresses; last error: UNKNOWN: ipv4:127.0.0.1:36157: Failed to connect to remote host: Connection refused", grpc_status:14, created_time:"2024-03-08T19:30:01.330340054+01:00"}"
>
E0308 19:30:01.359596 139757958891072 _internal.py:96] Error on request:
Traceback (most recent call last):
  File "/home/gcasadella/miniconda3/envs/openmmlab/lib/python3.8/site-packages/werkzeug/serving.py", line 362, in run_wsgi
    execute(self.server.app)
  File "/home/gcasadella/miniconda3/envs/openmmlab/lib/python3.8/site-packages/werkzeug/serving.py", line 323, in execute
    application_iter = app(environ, start_response)
  File "/home/gcasadella/miniconda3/envs/openmmlab/lib/python3.8/site-packages/tensorboard/backend/application.py", line 528, in __call__
    return self._app(environ, start_response)
  File "/home/gcasadella/miniconda3/envs/openmmlab/lib/python3.8/site-packages/tensorboard/backend/application.py", line 569, in wrapper
    return wsgi_app(environ, start_response)
  File "/home/gcasadella/miniconda3/envs/openmmlab/lib/python3.8/site-packages/tensorboard/backend/security_validator.py", line 91, in __call__
    return self._application(environ, start_response_proxy)
  File "/home/gcasadella/miniconda3/envs/openmmlab/lib/python3.8/site-packages/tensorboard/backend/path_prefix.py", line 68, in __call__
    return self._application(environ, start_response)
  File "/home/gcasadella/miniconda3/envs/openmmlab/lib/python3.8/site-packages/tensorboard/backend/experiment_id.py", line 73, in __call__
    return self._application(environ, start_response)
  File "/home/gcasadella/miniconda3/envs/openmmlab/lib/python3.8/site-packages/tensorboard/backend/empty_path_redirect.py", line 43, in __call__
    return self._application(environ, start_response)
  File "/home/gcasadella/miniconda3/envs/openmmlab/lib/python3.8/site-packages/tensorboard/backend/client_feature_flags.py", line 55, in __call__
    return self._application(environ, start_response)
  File "/home/gcasadella/miniconda3/envs/openmmlab/lib/python3.8/site-packages/tensorboard/backend/auth_context_middleware.py", line 38, in __call__
    return self._application(environ, start_response)
  File "/home/gcasadella/miniconda3/envs/openmmlab/lib/python3.8/site-packages/tensorboard/backend/application.py", line 551, in _route_request
    return self.exact_routes[clean_path](environ, start_response)
  File "/home/gcasadella/miniconda3/envs/openmmlab/lib/python3.8/site-packages/werkzeug/wrappers/request.py", line 189, in application
    resp = f(*args[:-2] + (request,))
  File "/home/gcasadella/miniconda3/envs/openmmlab/lib/python3.8/site-packages/tensorboard/plugins/core/core_plugin.py", line 183, in _serve_environment
    md = self._data_provider.experiment_metadata(
  File "/home/gcasadella/miniconda3/envs/openmmlab/lib/python3.8/site-packages/tensorboard/data/grpc_provider.py", line 56, in experiment_metadata
    res = self._stub.GetExperiment(req)
  File "/home/gcasadella/miniconda3/envs/openmmlab/lib/python3.8/site-packages/grpc/_channel.py", line 1160, in __call__
    return _end_unary_response_blocking(state, call, False, None)
  File "/home/gcasadella/miniconda3/envs/openmmlab/lib/python3.8/site-packages/grpc/_channel.py", line 1003, in _end_unary_response_blocking
    raise _InactiveRpcError(state)  # pytype: disable=not-instantiable
grpc._channel._InactiveRpcError: <_InactiveRpcError of RPC that terminated with:
	status = StatusCode.UNAVAILABLE
	details = "failed to connect to all addresses; last error: UNKNOWN: ipv4:127.0.0.1:36157: Failed to connect to remote host: Connection refused"
	debug_error_string = "UNKNOWN:Error received from peer  {created_time:"2024-03-08T19:30:01.342696062+01:00", grpc_status:14, grpc_message:"failed to connect to all addresses; last error: UNKNOWN: ipv4:127.0.0.1:36157: Failed to connect to remote host: Connection refused"}"
>
E0308 19:30:01.360447 139757950498368 _internal.py:96] Error on request:
Traceback (most recent call last):
  File "/home/gcasadella/miniconda3/envs/openmmlab/lib/python3.8/site-packages/werkzeug/serving.py", line 362, in run_wsgi
    execute(self.server.app)
  File "/home/gcasadella/miniconda3/envs/openmmlab/lib/python3.8/site-packages/werkzeug/serving.py", line 323, in execute
    application_iter = app(environ, start_response)
  File "/home/gcasadella/miniconda3/envs/openmmlab/lib/python3.8/site-packages/tensorboard/backend/application.py", line 528, in __call__
    return self._app(environ, start_response)
  File "/home/gcasadella/miniconda3/envs/openmmlab/lib/python3.8/site-packages/tensorboard/backend/application.py", line 569, in wrapper
    return wsgi_app(environ, start_response)
  File "/home/gcasadella/miniconda3/envs/openmmlab/lib/python3.8/site-packages/tensorboard/backend/security_validator.py", line 91, in __call__
    return self._application(environ, start_response_proxy)
  File "/home/gcasadella/miniconda3/envs/openmmlab/lib/python3.8/site-packages/tensorboard/backend/path_prefix.py", line 68, in __call__
    return self._application(environ, start_response)
  File "/home/gcasadella/miniconda3/envs/openmmlab/lib/python3.8/site-packages/tensorboard/backend/experiment_id.py", line 73, in __call__
    return self._application(environ, start_response)
  File "/home/gcasadella/miniconda3/envs/openmmlab/lib/python3.8/site-packages/tensorboard/backend/empty_path_redirect.py", line 43, in __call__
    return self._application(environ, start_response)
  File "/home/gcasadella/miniconda3/envs/openmmlab/lib/python3.8/site-packages/tensorboard/backend/client_feature_flags.py", line 55, in __call__
    return self._application(environ, start_response)
  File "/home/gcasadella/miniconda3/envs/openmmlab/lib/python3.8/site-packages/tensorboard/backend/auth_context_middleware.py", line 38, in __call__
    return self._application(environ, start_response)
  File "/home/gcasadella/miniconda3/envs/openmmlab/lib/python3.8/site-packages/tensorboard/backend/application.py", line 551, in _route_request
    return self.exact_routes[clean_path](environ, start_response)
  File "/home/gcasadella/miniconda3/envs/openmmlab/lib/python3.8/site-packages/werkzeug/wrappers/request.py", line 189, in application
    resp = f(*args[:-2] + (request,))
  File "/home/gcasadella/miniconda3/envs/openmmlab/lib/python3.8/site-packages/tensorboard/plugins/hparams/hparams_plugin.py", line 123, in get_experiment_route
    get_experiment.Handler(
  File "/home/gcasadella/miniconda3/envs/openmmlab/lib/python3.8/site-packages/tensorboard/plugins/hparams/get_experiment.py", line 43, in run
    self._backend_context.hparams_metadata(
  File "/home/gcasadella/miniconda3/envs/openmmlab/lib/python3.8/site-packages/tensorboard/plugins/hparams/backend_context.py", line 108, in hparams_metadata
    self._tb_context.data_provider.list_tensors(
  File "/home/gcasadella/miniconda3/envs/openmmlab/lib/python3.8/contextlib.py", line 75, in inner
    return func(*args, **kwds)
  File "/home/gcasadella/miniconda3/envs/openmmlab/lib/python3.8/site-packages/tensorboard/data/grpc_provider.py", line 162, in list_tensors
    res = self._stub.ListTensors(req)
  File "/home/gcasadella/miniconda3/envs/openmmlab/lib/python3.8/site-packages/grpc/_channel.py", line 1160, in __call__
    return _end_unary_response_blocking(state, call, False, None)
  File "/home/gcasadella/miniconda3/envs/openmmlab/lib/python3.8/site-packages/grpc/_channel.py", line 1003, in _end_unary_response_blocking
    raise _InactiveRpcError(state)  # pytype: disable=not-instantiable
grpc._channel._InactiveRpcError: <_InactiveRpcError of RPC that terminated with:
	status = StatusCode.UNAVAILABLE
	details = "failed to connect to all addresses; last error: UNKNOWN: ipv4:127.0.0.1:36157: Failed to connect to remote host: Connection refused"
	debug_error_string = "UNKNOWN:Error received from peer  {grpc_message:"failed to connect to all addresses; last error: UNKNOWN: ipv4:127.0.0.1:36157: Failed to connect to remote host: Connection refused", grpc_status:14, created_time:"2024-03-08T19:30:01.331972983+01:00"}"
>
E0308 19:30:01.384181 139758000838208 _internal.py:96] Error on request:
Traceback (most recent call last):
  File "/home/gcasadella/miniconda3/envs/openmmlab/lib/python3.8/site-packages/werkzeug/serving.py", line 362, in run_wsgi
    execute(self.server.app)
  File "/home/gcasadella/miniconda3/envs/openmmlab/lib/python3.8/site-packages/werkzeug/serving.py", line 323, in execute
    application_iter = app(environ, start_response)
  File "/home/gcasadella/miniconda3/envs/openmmlab/lib/python3.8/site-packages/tensorboard/backend/application.py", line 528, in __call__
    return self._app(environ, start_response)
  File "/home/gcasadella/miniconda3/envs/openmmlab/lib/python3.8/site-packages/tensorboard/backend/application.py", line 569, in wrapper
    return wsgi_app(environ, start_response)
  File "/home/gcasadella/miniconda3/envs/openmmlab/lib/python3.8/site-packages/tensorboard/backend/security_validator.py", line 91, in __call__
    return self._application(environ, start_response_proxy)
  File "/home/gcasadella/miniconda3/envs/openmmlab/lib/python3.8/site-packages/tensorboard/backend/path_prefix.py", line 68, in __call__
    return self._application(environ, start_response)
  File "/home/gcasadella/miniconda3/envs/openmmlab/lib/python3.8/site-packages/tensorboard/backend/experiment_id.py", line 73, in __call__
    return self._application(environ, start_response)
  File "/home/gcasadella/miniconda3/envs/openmmlab/lib/python3.8/site-packages/tensorboard/backend/empty_path_redirect.py", line 43, in __call__
    return self._application(environ, start_response)
  File "/home/gcasadella/miniconda3/envs/openmmlab/lib/python3.8/site-packages/tensorboard/backend/client_feature_flags.py", line 55, in __call__
    return self._application(environ, start_response)
  File "/home/gcasadella/miniconda3/envs/openmmlab/lib/python3.8/site-packages/tensorboard/backend/auth_context_middleware.py", line 38, in __call__
    return self._application(environ, start_response)
  File "/home/gcasadella/miniconda3/envs/openmmlab/lib/python3.8/site-packages/tensorboard/backend/application.py", line 551, in _route_request
    return self.exact_routes[clean_path](environ, start_response)
  File "/home/gcasadella/miniconda3/envs/openmmlab/lib/python3.8/site-packages/werkzeug/wrappers/request.py", line 189, in application
    resp = f(*args[:-2] + (request,))
  File "/home/gcasadella/miniconda3/envs/openmmlab/lib/python3.8/site-packages/tensorboard/plugins/core/core_plugin.py", line 183, in _serve_environment
    md = self._data_provider.experiment_metadata(
  File "/home/gcasadella/miniconda3/envs/openmmlab/lib/python3.8/site-packages/tensorboard/data/grpc_provider.py", line 56, in experiment_metadata
    res = self._stub.GetExperiment(req)
  File "/home/gcasadella/miniconda3/envs/openmmlab/lib/python3.8/site-packages/grpc/_channel.py", line 1160, in __call__
    return _end_unary_response_blocking(state, call, False, None)
  File "/home/gcasadella/miniconda3/envs/openmmlab/lib/python3.8/site-packages/grpc/_channel.py", line 1003, in _end_unary_response_blocking
    raise _InactiveRpcError(state)  # pytype: disable=not-instantiable
grpc._channel._InactiveRpcError: <_InactiveRpcError of RPC that terminated with:
	status = StatusCode.UNAVAILABLE
	details = "failed to connect to all addresses; last error: UNKNOWN: ipv4:127.0.0.1:36157: Failed to connect to remote host: Connection refused"
	debug_error_string = "UNKNOWN:Error received from peer  {created_time:"2024-03-08T19:30:01.379476868+01:00", grpc_status:14, grpc_message:"failed to connect to all addresses; last error: UNKNOWN: ipv4:127.0.0.1:36157: Failed to connect to remote host: Connection refused"}"
>
E0308 19:30:01.384563 139757958891072 _internal.py:96] Error on request:
Traceback (most recent call last):
  File "/home/gcasadella/miniconda3/envs/openmmlab/lib/python3.8/site-packages/werkzeug/serving.py", line 362, in run_wsgi
    execute(self.server.app)
  File "/home/gcasadella/miniconda3/envs/openmmlab/lib/python3.8/site-packages/werkzeug/serving.py", line 323, in execute
    application_iter = app(environ, start_response)
  File "/home/gcasadella/miniconda3/envs/openmmlab/lib/python3.8/site-packages/tensorboard/backend/application.py", line 528, in __call__
    return self._app(environ, start_response)
  File "/home/gcasadella/miniconda3/envs/openmmlab/lib/python3.8/site-packages/tensorboard/backend/application.py", line 569, in wrapper
    return wsgi_app(environ, start_response)
  File "/home/gcasadella/miniconda3/envs/openmmlab/lib/python3.8/site-packages/tensorboard/backend/security_validator.py", line 91, in __call__
    return self._application(environ, start_response_proxy)
  File "/home/gcasadella/miniconda3/envs/openmmlab/lib/python3.8/site-packages/tensorboard/backend/path_prefix.py", line 68, in __call__
    return self._application(environ, start_response)
  File "/home/gcasadella/miniconda3/envs/openmmlab/lib/python3.8/site-packages/tensorboard/backend/experiment_id.py", line 73, in __call__
    return self._application(environ, start_response)
  File "/home/gcasadella/miniconda3/envs/openmmlab/lib/python3.8/site-packages/tensorboard/backend/empty_path_redirect.py", line 43, in __call__
    return self._application(environ, start_response)
  File "/home/gcasadella/miniconda3/envs/openmmlab/lib/python3.8/site-packages/tensorboard/backend/client_feature_flags.py", line 55, in __call__
    return self._application(environ, start_response)
  File "/home/gcasadella/miniconda3/envs/openmmlab/lib/python3.8/site-packages/tensorboard/backend/auth_context_middleware.py", line 38, in __call__
    return self._application(environ, start_response)
  File "/home/gcasadella/miniconda3/envs/openmmlab/lib/python3.8/site-packages/tensorboard/backend/application.py", line 551, in _route_request
    return self.exact_routes[clean_path](environ, start_response)
  File "/home/gcasadella/miniconda3/envs/openmmlab/lib/python3.8/site-packages/werkzeug/wrappers/request.py", line 189, in application
    resp = f(*args[:-2] + (request,))
  File "/home/gcasadella/miniconda3/envs/openmmlab/lib/python3.8/site-packages/tensorboard/plugins/core/core_plugin.py", line 257, in _serve_runs
    self._data_provider.list_runs(ctx, experiment_id=experiment),
  File "/home/gcasadella/miniconda3/envs/openmmlab/lib/python3.8/site-packages/tensorboard/data/grpc_provider.py", line 76, in list_runs
    res = self._stub.ListRuns(req)
  File "/home/gcasadella/miniconda3/envs/openmmlab/lib/python3.8/site-packages/grpc/_channel.py", line 1160, in __call__
    return _end_unary_response_blocking(state, call, False, None)
  File "/home/gcasadella/miniconda3/envs/openmmlab/lib/python3.8/site-packages/grpc/_channel.py", line 1003, in _end_unary_response_blocking
    raise _InactiveRpcError(state)  # pytype: disable=not-instantiable
grpc._channel._InactiveRpcError: <_InactiveRpcError of RPC that terminated with:
	status = StatusCode.UNAVAILABLE
	details = "failed to connect to all addresses; last error: UNKNOWN: ipv4:127.0.0.1:36157: Failed to connect to remote host: Connection refused"
	debug_error_string = "UNKNOWN:Error received from peer  {grpc_message:"failed to connect to all addresses; last error: UNKNOWN: ipv4:127.0.0.1:36157: Failed to connect to remote host: Connection refused", grpc_status:14, created_time:"2024-03-08T19:30:01.378922251+01:00"}"
>
E0308 19:30:01.390575 139757958891072 _internal.py:96] Error on request:
Traceback (most recent call last):
  File "/home/gcasadella/miniconda3/envs/openmmlab/lib/python3.8/site-packages/werkzeug/serving.py", line 362, in run_wsgi
    execute(self.server.app)
  File "/home/gcasadella/miniconda3/envs/openmmlab/lib/python3.8/site-packages/werkzeug/serving.py", line 323, in execute
    application_iter = app(environ, start_response)
  File "/home/gcasadella/miniconda3/envs/openmmlab/lib/python3.8/site-packages/tensorboard/backend/application.py", line 528, in __call__
    return self._app(environ, start_response)
  File "/home/gcasadella/miniconda3/envs/openmmlab/lib/python3.8/site-packages/tensorboard/backend/application.py", line 569, in wrapper
    return wsgi_app(environ, start_response)
  File "/home/gcasadella/miniconda3/envs/openmmlab/lib/python3.8/site-packages/tensorboard/backend/security_validator.py", line 91, in __call__
    return self._application(environ, start_response_proxy)
  File "/home/gcasadella/miniconda3/envs/openmmlab/lib/python3.8/site-packages/tensorboard/backend/path_prefix.py", line 68, in __call__
    return self._application(environ, start_response)
  File "/home/gcasadella/miniconda3/envs/openmmlab/lib/python3.8/site-packages/tensorboard/backend/experiment_id.py", line 73, in __call__
    return self._application(environ, start_response)
  File "/home/gcasadella/miniconda3/envs/openmmlab/lib/python3.8/site-packages/tensorboard/backend/empty_path_redirect.py", line 43, in __call__
    return self._application(environ, start_response)
  File "/home/gcasadella/miniconda3/envs/openmmlab/lib/python3.8/site-packages/tensorboard/backend/client_feature_flags.py", line 55, in __call__
    return self._application(environ, start_response)
  File "/home/gcasadella/miniconda3/envs/openmmlab/lib/python3.8/site-packages/tensorboard/backend/auth_context_middleware.py", line 38, in __call__
    return self._application(environ, start_response)
  File "/home/gcasadella/miniconda3/envs/openmmlab/lib/python3.8/site-packages/tensorboard/backend/application.py", line 551, in _route_request
    return self.exact_routes[clean_path](environ, start_response)
  File "/home/gcasadella/miniconda3/envs/openmmlab/lib/python3.8/site-packages/werkzeug/wrappers/request.py", line 189, in application
    resp = f(*args[:-2] + (request,))
  File "/home/gcasadella/miniconda3/envs/openmmlab/lib/python3.8/site-packages/tensorboard/plugins/core/core_plugin.py", line 257, in _serve_runs
    self._data_provider.list_runs(ctx, experiment_id=experiment),
  File "/home/gcasadella/miniconda3/envs/openmmlab/lib/python3.8/site-packages/tensorboard/data/grpc_provider.py", line 76, in list_runs
    res = self._stub.ListRuns(req)
  File "/home/gcasadella/miniconda3/envs/openmmlab/lib/python3.8/site-packages/grpc/_channel.py", line 1160, in __call__
    return _end_unary_response_blocking(state, call, False, None)
  File "/home/gcasadella/miniconda3/envs/openmmlab/lib/python3.8/site-packages/grpc/_channel.py", line 1003, in _end_unary_response_blocking
    raise _InactiveRpcError(state)  # pytype: disable=not-instantiable
grpc._channel._InactiveRpcError: <_InactiveRpcError of RPC that terminated with:
	status = StatusCode.UNAVAILABLE
	details = "failed to connect to all addresses; last error: UNKNOWN: ipv4:127.0.0.1:36157: Failed to connect to remote host: Connection refused"
	debug_error_string = "UNKNOWN:Error received from peer  {grpc_message:"failed to connect to all addresses; last error: UNKNOWN: ipv4:127.0.0.1:36157: Failed to connect to remote host: Connection refused", grpc_status:14, created_time:"2024-03-08T19:30:01.38670019+01:00"}"
>
E0308 19:30:01.391652 139758000838208 _internal.py:96] Error on request:
Traceback (most recent call last):
  File "/home/gcasadella/miniconda3/envs/openmmlab/lib/python3.8/site-packages/werkzeug/serving.py", line 362, in run_wsgi
    execute(self.server.app)
  File "/home/gcasadella/miniconda3/envs/openmmlab/lib/python3.8/site-packages/werkzeug/serving.py", line 323, in execute
    application_iter = app(environ, start_response)
  File "/home/gcasadella/miniconda3/envs/openmmlab/lib/python3.8/site-packages/tensorboard/backend/application.py", line 528, in __call__
    return self._app(environ, start_response)
  File "/home/gcasadella/miniconda3/envs/openmmlab/lib/python3.8/site-packages/tensorboard/backend/application.py", line 569, in wrapper
    return wsgi_app(environ, start_response)
  File "/home/gcasadella/miniconda3/envs/openmmlab/lib/python3.8/site-packages/tensorboard/backend/security_validator.py", line 91, in __call__
    return self._application(environ, start_response_proxy)
  File "/home/gcasadella/miniconda3/envs/openmmlab/lib/python3.8/site-packages/tensorboard/backend/path_prefix.py", line 68, in __call__
    return self._application(environ, start_response)
  File "/home/gcasadella/miniconda3/envs/openmmlab/lib/python3.8/site-packages/tensorboard/backend/experiment_id.py", line 73, in __call__
    return self._application(environ, start_response)
  File "/home/gcasadella/miniconda3/envs/openmmlab/lib/python3.8/site-packages/tensorboard/backend/empty_path_redirect.py", line 43, in __call__
    return self._application(environ, start_response)
  File "/home/gcasadella/miniconda3/envs/openmmlab/lib/python3.8/site-packages/tensorboard/backend/client_feature_flags.py", line 55, in __call__
    return self._application(environ, start_response)
  File "/home/gcasadella/miniconda3/envs/openmmlab/lib/python3.8/site-packages/tensorboard/backend/auth_context_middleware.py", line 38, in __call__
    return self._application(environ, start_response)
  File "/home/gcasadella/miniconda3/envs/openmmlab/lib/python3.8/site-packages/tensorboard/backend/application.py", line 551, in _route_request
    return self.exact_routes[clean_path](environ, start_response)
  File "/home/gcasadella/miniconda3/envs/openmmlab/lib/python3.8/site-packages/werkzeug/wrappers/request.py", line 189, in application
    resp = f(*args[:-2] + (request,))
  File "/home/gcasadella/miniconda3/envs/openmmlab/lib/python3.8/site-packages/tensorboard/plugins/core/core_plugin.py", line 183, in _serve_environment
    md = self._data_provider.experiment_metadata(
  File "/home/gcasadella/miniconda3/envs/openmmlab/lib/python3.8/site-packages/tensorboard/data/grpc_provider.py", line 56, in experiment_metadata
    res = self._stub.GetExperiment(req)
  File "/home/gcasadella/miniconda3/envs/openmmlab/lib/python3.8/site-packages/grpc/_channel.py", line 1160, in __call__
    return _end_unary_response_blocking(state, call, False, None)
  File "/home/gcasadella/miniconda3/envs/openmmlab/lib/python3.8/site-packages/grpc/_channel.py", line 1003, in _end_unary_response_blocking
    raise _InactiveRpcError(state)  # pytype: disable=not-instantiable
grpc._channel._InactiveRpcError: <_InactiveRpcError of RPC that terminated with:
	status = StatusCode.UNAVAILABLE
	details = "failed to connect to all addresses; last error: UNKNOWN: ipv4:127.0.0.1:36157: Failed to connect to remote host: Connection refused"
	debug_error_string = "UNKNOWN:Error received from peer  {grpc_message:"failed to connect to all addresses; last error: UNKNOWN: ipv4:127.0.0.1:36157: Failed to connect to remote host: Connection refused", grpc_status:14, created_time:"2024-03-08T19:30:01.387703482+01:00"}"
>
E0308 19:30:01.398238 139758000838208 _internal.py:96] Error on request:
Traceback (most recent call last):
  File "/home/gcasadella/miniconda3/envs/openmmlab/lib/python3.8/site-packages/werkzeug/serving.py", line 362, in run_wsgi
    execute(self.server.app)
  File "/home/gcasadella/miniconda3/envs/openmmlab/lib/python3.8/site-packages/werkzeug/serving.py", line 323, in execute
    application_iter = app(environ, start_response)
  File "/home/gcasadella/miniconda3/envs/openmmlab/lib/python3.8/site-packages/tensorboard/backend/application.py", line 528, in __call__
    return self._app(environ, start_response)
  File "/home/gcasadella/miniconda3/envs/openmmlab/lib/python3.8/site-packages/tensorboard/backend/application.py", line 569, in wrapper
    return wsgi_app(environ, start_response)
  File "/home/gcasadella/miniconda3/envs/openmmlab/lib/python3.8/site-packages/tensorboard/backend/security_validator.py", line 91, in __call__
    return self._application(environ, start_response_proxy)
  File "/home/gcasadella/miniconda3/envs/openmmlab/lib/python3.8/site-packages/tensorboard/backend/path_prefix.py", line 68, in __call__
    return self._application(environ, start_response)
  File "/home/gcasadella/miniconda3/envs/openmmlab/lib/python3.8/site-packages/tensorboard/backend/experiment_id.py", line 73, in __call__
    return self._application(environ, start_response)
  File "/home/gcasadella/miniconda3/envs/openmmlab/lib/python3.8/site-packages/tensorboard/backend/empty_path_redirect.py", line 43, in __call__
    return self._application(environ, start_response)
  File "/home/gcasadella/miniconda3/envs/openmmlab/lib/python3.8/site-packages/tensorboard/backend/client_feature_flags.py", line 55, in __call__
    return self._application(environ, start_response)
  File "/home/gcasadella/miniconda3/envs/openmmlab/lib/python3.8/site-packages/tensorboard/backend/auth_context_middleware.py", line 38, in __call__
    return self._application(environ, start_response)
  File "/home/gcasadella/miniconda3/envs/openmmlab/lib/python3.8/site-packages/tensorboard/backend/application.py", line 551, in _route_request
    return self.exact_routes[clean_path](environ, start_response)
  File "/home/gcasadella/miniconda3/envs/openmmlab/lib/python3.8/site-packages/werkzeug/wrappers/request.py", line 189, in application
    resp = f(*args[:-2] + (request,))
  File "/home/gcasadella/miniconda3/envs/openmmlab/lib/python3.8/site-packages/tensorboard/plugins/core/core_plugin.py", line 257, in _serve_runs
    self._data_provider.list_runs(ctx, experiment_id=experiment),
  File "/home/gcasadella/miniconda3/envs/openmmlab/lib/python3.8/site-packages/tensorboard/data/grpc_provider.py", line 76, in list_runs
    res = self._stub.ListRuns(req)
  File "/home/gcasadella/miniconda3/envs/openmmlab/lib/python3.8/site-packages/grpc/_channel.py", line 1160, in __call__
    return _end_unary_response_blocking(state, call, False, None)
  File "/home/gcasadella/miniconda3/envs/openmmlab/lib/python3.8/site-packages/grpc/_channel.py", line 1003, in _end_unary_response_blocking
    raise _InactiveRpcError(state)  # pytype: disable=not-instantiable
grpc._channel._InactiveRpcError: <_InactiveRpcError of RPC that terminated with:
	status = StatusCode.UNAVAILABLE
	details = "failed to connect to all addresses; last error: UNKNOWN: ipv4:127.0.0.1:36157: Failed to connect to remote host: Connection refused"
	debug_error_string = "UNKNOWN:Error received from peer  {grpc_message:"failed to connect to all addresses; last error: UNKNOWN: ipv4:127.0.0.1:36157: Failed to connect to remote host: Connection refused", grpc_status:14, created_time:"2024-03-08T19:30:01.394517744+01:00"}"
>
E0308 19:30:01.399796 139757958891072 _internal.py:96] Error on request:
Traceback (most recent call last):
  File "/home/gcasadella/miniconda3/envs/openmmlab/lib/python3.8/site-packages/werkzeug/serving.py", line 362, in run_wsgi
    execute(self.server.app)
  File "/home/gcasadella/miniconda3/envs/openmmlab/lib/python3.8/site-packages/werkzeug/serving.py", line 323, in execute
    application_iter = app(environ, start_response)
  File "/home/gcasadella/miniconda3/envs/openmmlab/lib/python3.8/site-packages/tensorboard/backend/application.py", line 528, in __call__
    return self._app(environ, start_response)
  File "/home/gcasadella/miniconda3/envs/openmmlab/lib/python3.8/site-packages/tensorboard/backend/application.py", line 569, in wrapper
    return wsgi_app(environ, start_response)
  File "/home/gcasadella/miniconda3/envs/openmmlab/lib/python3.8/site-packages/tensorboard/backend/security_validator.py", line 91, in __call__
    return self._application(environ, start_response_proxy)
  File "/home/gcasadella/miniconda3/envs/openmmlab/lib/python3.8/site-packages/tensorboard/backend/path_prefix.py", line 68, in __call__
    return self._application(environ, start_response)
  File "/home/gcasadella/miniconda3/envs/openmmlab/lib/python3.8/site-packages/tensorboard/backend/experiment_id.py", line 73, in __call__
    return self._application(environ, start_response)
  File "/home/gcasadella/miniconda3/envs/openmmlab/lib/python3.8/site-packages/tensorboard/backend/empty_path_redirect.py", line 43, in __call__
    return self._application(environ, start_response)
  File "/home/gcasadella/miniconda3/envs/openmmlab/lib/python3.8/site-packages/tensorboard/backend/client_feature_flags.py", line 55, in __call__
    return self._application(environ, start_response)
  File "/home/gcasadella/miniconda3/envs/openmmlab/lib/python3.8/site-packages/tensorboard/backend/auth_context_middleware.py", line 38, in __call__
    return self._application(environ, start_response)
  File "/home/gcasadella/miniconda3/envs/openmmlab/lib/python3.8/site-packages/tensorboard/backend/application.py", line 551, in _route_request
    return self.exact_routes[clean_path](environ, start_response)
  File "/home/gcasadella/miniconda3/envs/openmmlab/lib/python3.8/site-packages/werkzeug/wrappers/request.py", line 189, in application
    resp = f(*args[:-2] + (request,))
  File "/home/gcasadella/miniconda3/envs/openmmlab/lib/python3.8/site-packages/tensorboard/plugins/core/core_plugin.py", line 183, in _serve_environment
    md = self._data_provider.experiment_metadata(
  File "/home/gcasadella/miniconda3/envs/openmmlab/lib/python3.8/site-packages/tensorboard/data/grpc_provider.py", line 56, in experiment_metadata
    res = self._stub.GetExperiment(req)
  File "/home/gcasadella/miniconda3/envs/openmmlab/lib/python3.8/site-packages/grpc/_channel.py", line 1160, in __call__
    return _end_unary_response_blocking(state, call, False, None)
  File "/home/gcasadella/miniconda3/envs/openmmlab/lib/python3.8/site-packages/grpc/_channel.py", line 1003, in _end_unary_response_blocking
    raise _InactiveRpcError(state)  # pytype: disable=not-instantiable
grpc._channel._InactiveRpcError: <_InactiveRpcError of RPC that terminated with:
	status = StatusCode.UNAVAILABLE
	details = "failed to connect to all addresses; last error: UNKNOWN: ipv4:127.0.0.1:36157: Failed to connect to remote host: Connection refused"
	debug_error_string = "UNKNOWN:Error received from peer  {created_time:"2024-03-08T19:30:01.395572978+01:00", grpc_status:14, grpc_message:"failed to connect to all addresses; last error: UNKNOWN: ipv4:127.0.0.1:36157: Failed to connect to remote host: Connection refused"}"
>
TensorBoard caught SIGTERM; exiting...
2024-03-08 19:31:14.561282: I tensorflow/tsl/cuda/cudart_stub.cc:28] Could not find cuda drivers on your machine, GPU will not be used.
2024-03-08 19:31:14.615891: I tensorflow/tsl/cuda/cudart_stub.cc:28] Could not find cuda drivers on your machine, GPU will not be used.
2024-03-08 19:31:14.616532: I tensorflow/core/platform/cpu_feature_guard.cc:182] This TensorFlow binary is optimized to use available CPU instructions in performance-critical operations.
To enable the following instructions: AVX2 FMA, in other operations, rebuild TensorFlow with the appropriate compiler flags.
2024-03-08 19:31:15.504242: W tensorflow/compiler/tf2tensorrt/utils/py_utils.cc:38] TF-TRT Warning: Could not find TensorRT
2024-03-08 19:31:16.854721: W tensorflow/core/common_runtime/gpu/gpu_device.cc:1960] Cannot dlopen some GPU libraries. Please make sure the missing libraries mentioned above are installed properly if you would like to use GPU. Follow the guide at https://www.tensorflow.org/install/gpu for how to download and setup the required libraries for your platform.
Skipping registering GPU devices...

NOTE: Using experimental fast data loading logic. To disable, pass
    "--load_fast=false" and report issues on GitHub. More details:
    https://github.com/tensorflow/tensorboard/issues/4784

TensorBoard 2.13.0 at http://cudahpc09:6006/ (Press CTRL+C to quit)
2024-03-11 19:00:07.357990: I tensorflow/tsl/cuda/cudart_stub.cc:28] Could not find cuda drivers on your machine, GPU will not be used.
2024-03-11 19:00:07.409160: I tensorflow/tsl/cuda/cudart_stub.cc:28] Could not find cuda drivers on your machine, GPU will not be used.
2024-03-11 19:00:07.409782: I tensorflow/core/platform/cpu_feature_guard.cc:182] This TensorFlow binary is optimized to use available CPU instructions in performance-critical operations.
To enable the following instructions: AVX2 FMA, in other operations, rebuild TensorFlow with the appropriate compiler flags.
2024-03-11 19:00:08.216864: W tensorflow/compiler/tf2tensorrt/utils/py_utils.cc:38] TF-TRT Warning: Could not find TensorRT
2024-03-11 19:00:09.508631: W tensorflow/core/common_runtime/gpu/gpu_device.cc:1960] Cannot dlopen some GPU libraries. Please make sure the missing libraries mentioned above are installed properly if you would like to use GPU. Follow the guide at https://www.tensorflow.org/install/gpu for how to download and setup the required libraries for your platform.
Skipping registering GPU devices...

NOTE: Using experimental fast data loading logic. To disable, pass
    "--load_fast=false" and report issues on GitHub. More details:
    https://github.com/tensorflow/tensorboard/issues/4784

TensorBoard 2.13.0 at http://cudahpc09:6007/ (Press CTRL+C to quit)
2024-03-11 19:01:28.959309: I tensorflow/tsl/cuda/cudart_stub.cc:28] Could not find cuda drivers on your machine, GPU will not be used.
2024-03-11 19:01:29.010157: I tensorflow/tsl/cuda/cudart_stub.cc:28] Could not find cuda drivers on your machine, GPU will not be used.
2024-03-11 19:01:29.010775: I tensorflow/core/platform/cpu_feature_guard.cc:182] This TensorFlow binary is optimized to use available CPU instructions in performance-critical operations.
To enable the following instructions: AVX2 FMA, in other operations, rebuild TensorFlow with the appropriate compiler flags.
2024-03-11 19:01:29.831161: W tensorflow/compiler/tf2tensorrt/utils/py_utils.cc:38] TF-TRT Warning: Could not find TensorRT
2024-03-11 19:01:31.017376: W tensorflow/core/common_runtime/gpu/gpu_device.cc:1960] Cannot dlopen some GPU libraries. Please make sure the missing libraries mentioned above are installed properly if you would like to use GPU. Follow the guide at https://www.tensorflow.org/install/gpu for how to download and setup the required libraries for your platform.
Skipping registering GPU devices...

NOTE: Using experimental fast data loading logic. To disable, pass
    "--load_fast=false" and report issues on GitHub. More details:
    https://github.com/tensorflow/tensorboard/issues/4784

TensorBoard 2.13.0 at http://cudahpc09:6008/ (Press CTRL+C to quit)
2024-03-13 16:31:19.384698: I tensorflow/tsl/cuda/cudart_stub.cc:28] Could not find cuda drivers on your machine, GPU will not be used.
2024-03-13 16:31:19.448078: I tensorflow/tsl/cuda/cudart_stub.cc:28] Could not find cuda drivers on your machine, GPU will not be used.
2024-03-13 16:31:19.448664: I tensorflow/core/platform/cpu_feature_guard.cc:182] This TensorFlow binary is optimized to use available CPU instructions in performance-critical operations.
To enable the following instructions: AVX2 FMA, in other operations, rebuild TensorFlow with the appropriate compiler flags.
2024-03-13 16:31:20.290129: W tensorflow/compiler/tf2tensorrt/utils/py_utils.cc:38] TF-TRT Warning: Could not find TensorRT
2024-03-13 16:31:22.037507: W tensorflow/core/common_runtime/gpu/gpu_device.cc:1960] Cannot dlopen some GPU libraries. Please make sure the missing libraries mentioned above are installed properly if you would like to use GPU. Follow the guide at https://www.tensorflow.org/install/gpu for how to download and setup the required libraries for your platform.
Skipping registering GPU devices...

NOTE: Using experimental fast data loading logic. To disable, pass
    "--load_fast=false" and report issues on GitHub. More details:
    https://github.com/tensorflow/tensorboard/issues/4784

TensorBoard 2.13.0 at http://cudahpc09:6009/ (Press CTRL+C to quit)
2024-03-13 16:32:36.393947: I tensorflow/tsl/cuda/cudart_stub.cc:28] Could not find cuda drivers on your machine, GPU will not be used.
2024-03-13 16:32:36.448252: I tensorflow/tsl/cuda/cudart_stub.cc:28] Could not find cuda drivers on your machine, GPU will not be used.
2024-03-13 16:32:36.448883: I tensorflow/core/platform/cpu_feature_guard.cc:182] This TensorFlow binary is optimized to use available CPU instructions in performance-critical operations.
To enable the following instructions: AVX2 FMA, in other operations, rebuild TensorFlow with the appropriate compiler flags.
2024-03-13 16:32:37.360473: W tensorflow/compiler/tf2tensorrt/utils/py_utils.cc:38] TF-TRT Warning: Could not find TensorRT
2024-03-13 16:32:38.901217: W tensorflow/core/common_runtime/gpu/gpu_device.cc:1960] Cannot dlopen some GPU libraries. Please make sure the missing libraries mentioned above are installed properly if you would like to use GPU. Follow the guide at https://www.tensorflow.org/install/gpu for how to download and setup the required libraries for your platform.
Skipping registering GPU devices...

NOTE: Using experimental fast data loading logic. To disable, pass
    "--load_fast=false" and report issues on GitHub. More details:
    https://github.com/tensorflow/tensorboard/issues/4784

Address already in use
Port 6010 is in use by another program. Either identify and stop that program, or start the server with a different port.
2024-03-13 16:36:24.992971: I tensorflow/tsl/cuda/cudart_stub.cc:28] Could not find cuda drivers on your machine, GPU will not be used.
2024-03-13 16:36:25.046868: I tensorflow/tsl/cuda/cudart_stub.cc:28] Could not find cuda drivers on your machine, GPU will not be used.
2024-03-13 16:36:25.047588: I tensorflow/core/platform/cpu_feature_guard.cc:182] This TensorFlow binary is optimized to use available CPU instructions in performance-critical operations.
To enable the following instructions: AVX2 FMA, in other operations, rebuild TensorFlow with the appropriate compiler flags.
2024-03-13 16:36:25.920120: W tensorflow/compiler/tf2tensorrt/utils/py_utils.cc:38] TF-TRT Warning: Could not find TensorRT
2024-03-13 16:36:27.377239: W tensorflow/core/common_runtime/gpu/gpu_device.cc:1960] Cannot dlopen some GPU libraries. Please make sure the missing libraries mentioned above are installed properly if you would like to use GPU. Follow the guide at https://www.tensorflow.org/install/gpu for how to download and setup the required libraries for your platform.
Skipping registering GPU devices...

NOTE: Using experimental fast data loading logic. To disable, pass
    "--load_fast=false" and report issues on GitHub. More details:
    https://github.com/tensorflow/tensorboard/issues/4784

TensorBoard 2.13.0 at http://cudahpc09:6012/ (Press CTRL+C to quit)
TensorBoard caught SIGTERM; exiting...
2024-03-13 16:38:26.607395: I tensorflow/tsl/cuda/cudart_stub.cc:28] Could not find cuda drivers on your machine, GPU will not be used.
2024-03-13 16:38:26.663662: I tensorflow/tsl/cuda/cudart_stub.cc:28] Could not find cuda drivers on your machine, GPU will not be used.
2024-03-13 16:38:26.664333: I tensorflow/core/platform/cpu_feature_guard.cc:182] This TensorFlow binary is optimized to use available CPU instructions in performance-critical operations.
To enable the following instructions: AVX2 FMA, in other operations, rebuild TensorFlow with the appropriate compiler flags.
2024-03-13 16:38:27.568570: W tensorflow/compiler/tf2tensorrt/utils/py_utils.cc:38] TF-TRT Warning: Could not find TensorRT
2024-03-13 16:38:29.103774: W tensorflow/core/common_runtime/gpu/gpu_device.cc:1960] Cannot dlopen some GPU libraries. Please make sure the missing libraries mentioned above are installed properly if you would like to use GPU. Follow the guide at https://www.tensorflow.org/install/gpu for how to download and setup the required libraries for your platform.
Skipping registering GPU devices...

NOTE: Using experimental fast data loading logic. To disable, pass
    "--load_fast=false" and report issues on GitHub. More details:
    https://github.com/tensorflow/tensorboard/issues/4784

TensorBoard 2.13.0 at http://cudahpc09:6012/ (Press CTRL+C to quit)
TensorBoard caught SIGTERM; exiting...
TensorBoard caught SIGTERM; exiting...
2024-03-15 16:06:59.004223: I tensorflow/tsl/cuda/cudart_stub.cc:28] Could not find cuda drivers on your machine, GPU will not be used.
2024-03-15 16:06:59.049271: I tensorflow/tsl/cuda/cudart_stub.cc:28] Could not find cuda drivers on your machine, GPU will not be used.
2024-03-15 16:06:59.049829: I tensorflow/core/platform/cpu_feature_guard.cc:182] This TensorFlow binary is optimized to use available CPU instructions in performance-critical operations.
To enable the following instructions: AVX2 FMA, in other operations, rebuild TensorFlow with the appropriate compiler flags.
2024-03-15 16:06:59.908614: W tensorflow/compiler/tf2tensorrt/utils/py_utils.cc:38] TF-TRT Warning: Could not find TensorRT
2024-03-15 16:07:01.401581: W tensorflow/core/common_runtime/gpu/gpu_device.cc:1960] Cannot dlopen some GPU libraries. Please make sure the missing libraries mentioned above are installed properly if you would like to use GPU. Follow the guide at https://www.tensorflow.org/install/gpu for how to download and setup the required libraries for your platform.
Skipping registering GPU devices...

NOTE: Using experimental fast data loading logic. To disable, pass
    "--load_fast=false" and report issues on GitHub. More details:
    https://github.com/tensorflow/tensorboard/issues/4784

TensorBoard 2.13.0 at http://cudahpc09:6006/ (Press CTRL+C to quit)
TensorBoard caught SIGTERM; exiting...
TensorBoard caught SIGTERM; exiting...
TensorBoard caught SIGTERM; exiting...
TensorBoard caught SIGTERM; exiting...
2024-03-19 18:17:26.082804: I tensorflow/tsl/cuda/cudart_stub.cc:28] Could not find cuda drivers on your machine, GPU will not be used.
2024-03-19 18:17:28.400216: I tensorflow/tsl/cuda/cudart_stub.cc:28] Could not find cuda drivers on your machine, GPU will not be used.
2024-03-19 18:17:28.403182: I tensorflow/core/platform/cpu_feature_guard.cc:182] This TensorFlow binary is optimized to use available CPU instructions in performance-critical operations.
To enable the following instructions: AVX2 FMA, in other operations, rebuild TensorFlow with the appropriate compiler flags.
2024-03-19 18:17:30.814769: W tensorflow/compiler/tf2tensorrt/utils/py_utils.cc:38] TF-TRT Warning: Could not find TensorRT
2024-03-19 18:17:36.561964: W tensorflow/core/common_runtime/gpu/gpu_device.cc:1960] Cannot dlopen some GPU libraries. Please make sure the missing libraries mentioned above are installed properly if you would like to use GPU. Follow the guide at https://www.tensorflow.org/install/gpu for how to download and setup the required libraries for your platform.
Skipping registering GPU devices...

NOTE: Using experimental fast data loading logic. To disable, pass
    "--load_fast=false" and report issues on GitHub. More details:
    https://github.com/tensorflow/tensorboard/issues/4784

TensorBoard 2.13.0 at http://cudahpc09:6006/ (Press CTRL+C to quit)
2024-03-19 18:19:15.111177: I tensorflow/tsl/cuda/cudart_stub.cc:28] Could not find cuda drivers on your machine, GPU will not be used.
2024-03-19 18:19:15.156425: I tensorflow/tsl/cuda/cudart_stub.cc:28] Could not find cuda drivers on your machine, GPU will not be used.
2024-03-19 18:19:15.156975: I tensorflow/core/platform/cpu_feature_guard.cc:182] This TensorFlow binary is optimized to use available CPU instructions in performance-critical operations.
To enable the following instructions: AVX2 FMA, in other operations, rebuild TensorFlow with the appropriate compiler flags.
2024-03-19 18:19:15.868017: W tensorflow/compiler/tf2tensorrt/utils/py_utils.cc:38] TF-TRT Warning: Could not find TensorRT
2024-03-19 18:19:16.913958: W tensorflow/core/common_runtime/gpu/gpu_device.cc:1960] Cannot dlopen some GPU libraries. Please make sure the missing libraries mentioned above are installed properly if you would like to use GPU. Follow the guide at https://www.tensorflow.org/install/gpu for how to download and setup the required libraries for your platform.
Skipping registering GPU devices...

NOTE: Using experimental fast data loading logic. To disable, pass
    "--load_fast=false" and report issues on GitHub. More details:
    https://github.com/tensorflow/tensorboard/issues/4784

TensorBoard 2.13.0 at http://cudahpc09:6007/ (Press CTRL+C to quit)
2024-03-19 18:20:22.248640: I tensorflow/tsl/cuda/cudart_stub.cc:28] Could not find cuda drivers on your machine, GPU will not be used.
2024-03-19 18:20:22.294331: I tensorflow/tsl/cuda/cudart_stub.cc:28] Could not find cuda drivers on your machine, GPU will not be used.
2024-03-19 18:20:22.294889: I tensorflow/core/platform/cpu_feature_guard.cc:182] This TensorFlow binary is optimized to use available CPU instructions in performance-critical operations.
To enable the following instructions: AVX2 FMA, in other operations, rebuild TensorFlow with the appropriate compiler flags.
2024-03-19 18:20:23.004888: W tensorflow/compiler/tf2tensorrt/utils/py_utils.cc:38] TF-TRT Warning: Could not find TensorRT
2024-03-19 18:20:24.061275: W tensorflow/core/common_runtime/gpu/gpu_device.cc:1960] Cannot dlopen some GPU libraries. Please make sure the missing libraries mentioned above are installed properly if you would like to use GPU. Follow the guide at https://www.tensorflow.org/install/gpu for how to download and setup the required libraries for your platform.
Skipping registering GPU devices...

NOTE: Using experimental fast data loading logic. To disable, pass
    "--load_fast=false" and report issues on GitHub. More details:
    https://github.com/tensorflow/tensorboard/issues/4784

TensorBoard 2.13.0 at http://cudahpc09:6008/ (Press CTRL+C to quit)
2024-03-20 16:41:15.464425: I tensorflow/tsl/cuda/cudart_stub.cc:28] Could not find cuda drivers on your machine, GPU will not be used.
2024-03-20 16:41:15.542009: I tensorflow/tsl/cuda/cudart_stub.cc:28] Could not find cuda drivers on your machine, GPU will not be used.
2024-03-20 16:41:15.542700: I tensorflow/core/platform/cpu_feature_guard.cc:182] This TensorFlow binary is optimized to use available CPU instructions in performance-critical operations.
To enable the following instructions: AVX2 FMA, in other operations, rebuild TensorFlow with the appropriate compiler flags.
2024-03-20 16:41:16.950444: W tensorflow/compiler/tf2tensorrt/utils/py_utils.cc:38] TF-TRT Warning: Could not find TensorRT
2024-03-20 16:41:19.661097: W tensorflow/core/common_runtime/gpu/gpu_device.cc:1960] Cannot dlopen some GPU libraries. Please make sure the missing libraries mentioned above are installed properly if you would like to use GPU. Follow the guide at https://www.tensorflow.org/install/gpu for how to download and setup the required libraries for your platform.
Skipping registering GPU devices...

NOTE: Using experimental fast data loading logic. To disable, pass
    "--load_fast=false" and report issues on GitHub. More details:
    https://github.com/tensorflow/tensorboard/issues/4784

TensorBoard 2.13.0 at http://cudahpc09:6009/ (Press CTRL+C to quit)
2024-03-20 16:41:32.112464: I tensorflow/tsl/cuda/cudart_stub.cc:28] Could not find cuda drivers on your machine, GPU will not be used.
2024-03-20 16:41:32.165121: I tensorflow/tsl/cuda/cudart_stub.cc:28] Could not find cuda drivers on your machine, GPU will not be used.
2024-03-20 16:41:32.165767: I tensorflow/core/platform/cpu_feature_guard.cc:182] This TensorFlow binary is optimized to use available CPU instructions in performance-critical operations.
To enable the following instructions: AVX2 FMA, in other operations, rebuild TensorFlow with the appropriate compiler flags.
2024-03-20 16:41:33.012471: W tensorflow/compiler/tf2tensorrt/utils/py_utils.cc:38] TF-TRT Warning: Could not find TensorRT
2024-03-20 16:41:34.607380: W tensorflow/core/common_runtime/gpu/gpu_device.cc:1960] Cannot dlopen some GPU libraries. Please make sure the missing libraries mentioned above are installed properly if you would like to use GPU. Follow the guide at https://www.tensorflow.org/install/gpu for how to download and setup the required libraries for your platform.
Skipping registering GPU devices...

NOTE: Using experimental fast data loading logic. To disable, pass
    "--load_fast=false" and report issues on GitHub. More details:
    https://github.com/tensorflow/tensorboard/issues/4784

Address already in use
Port 6010 is in use by another program. Either identify and stop that program, or start the server with a different port.
2024-03-20 16:41:50.130276: I tensorflow/tsl/cuda/cudart_stub.cc:28] Could not find cuda drivers on your machine, GPU will not be used.
2024-03-20 16:41:50.188343: I tensorflow/tsl/cuda/cudart_stub.cc:28] Could not find cuda drivers on your machine, GPU will not be used.
2024-03-20 16:41:50.188952: I tensorflow/core/platform/cpu_feature_guard.cc:182] This TensorFlow binary is optimized to use available CPU instructions in performance-critical operations.
To enable the following instructions: AVX2 FMA, in other operations, rebuild TensorFlow with the appropriate compiler flags.
2024-03-20 16:41:51.071692: W tensorflow/compiler/tf2tensorrt/utils/py_utils.cc:38] TF-TRT Warning: Could not find TensorRT
2024-03-20 16:41:52.561237: W tensorflow/core/common_runtime/gpu/gpu_device.cc:1960] Cannot dlopen some GPU libraries. Please make sure the missing libraries mentioned above are installed properly if you would like to use GPU. Follow the guide at https://www.tensorflow.org/install/gpu for how to download and setup the required libraries for your platform.
Skipping registering GPU devices...

NOTE: Using experimental fast data loading logic. To disable, pass
    "--load_fast=false" and report issues on GitHub. More details:
    https://github.com/tensorflow/tensorboard/issues/4784

TensorBoard 2.13.0 at http://cudahpc09:6011/ (Press CTRL+C to quit)
2024-03-20 16:43:24.578393: I tensorflow/tsl/cuda/cudart_stub.cc:28] Could not find cuda drivers on your machine, GPU will not be used.
2024-03-20 16:43:24.629907: I tensorflow/tsl/cuda/cudart_stub.cc:28] Could not find cuda drivers on your machine, GPU will not be used.
2024-03-20 16:43:24.630525: I tensorflow/core/platform/cpu_feature_guard.cc:182] This TensorFlow binary is optimized to use available CPU instructions in performance-critical operations.
To enable the following instructions: AVX2 FMA, in other operations, rebuild TensorFlow with the appropriate compiler flags.
2024-03-20 16:43:25.511589: W tensorflow/compiler/tf2tensorrt/utils/py_utils.cc:38] TF-TRT Warning: Could not find TensorRT
2024-03-20 16:43:26.898456: W tensorflow/core/common_runtime/gpu/gpu_device.cc:1960] Cannot dlopen some GPU libraries. Please make sure the missing libraries mentioned above are installed properly if you would like to use GPU. Follow the guide at https://www.tensorflow.org/install/gpu for how to download and setup the required libraries for your platform.
Skipping registering GPU devices...

NOTE: Using experimental fast data loading logic. To disable, pass
    "--load_fast=false" and report issues on GitHub. More details:
    https://github.com/tensorflow/tensorboard/issues/4784

TensorBoard 2.13.0 at http://cudahpc09:6012/ (Press CTRL+C to quit)
TensorBoard caught SIGTERM; exiting...
TensorBoard caught SIGTERM; exiting...
TensorBoard caught SIGTERM; exiting...
TensorBoard caught SIGTERM; exiting...
TensorBoard caught SIGTERM; exiting...
TensorBoard caught SIGTERM; exiting...
E0405 12:52:45.077580226   63548 backup_poller.cc:113]                 run_poller: UNKNOWN:Timer list shutdown {created_time:"2024-04-05T12:52:45.077516943+02:00"}
2024-04-18 16:24:50.578291: I tensorflow/tsl/cuda/cudart_stub.cc:28] Could not find cuda drivers on your machine, GPU will not be used.
2024-04-18 16:24:50.735905: I tensorflow/tsl/cuda/cudart_stub.cc:28] Could not find cuda drivers on your machine, GPU will not be used.
2024-04-18 16:24:50.736916: I tensorflow/core/platform/cpu_feature_guard.cc:182] This TensorFlow binary is optimized to use available CPU instructions in performance-critical operations.
To enable the following instructions: AVX2 FMA, in other operations, rebuild TensorFlow with the appropriate compiler flags.
2024-04-18 16:24:52.646817: W tensorflow/compiler/tf2tensorrt/utils/py_utils.cc:38] TF-TRT Warning: Could not find TensorRT
usage: tensorboard [-h] [--helpfull] [--logdir PATH] [--logdir_spec PATH_SPEC]
                   [--host ADDR] [--bind_all] [--port PORT]
                   [--reuse_port BOOL] [--load_fast {false,auto,true}]
                   [--extra_data_server_flags EXTRA_DATA_SERVER_FLAGS]
                   [--grpc_creds_type {local,ssl,ssl_dev}]
                   [--grpc_data_provider PORT] [--purge_orphaned_data BOOL]
                   [--db URI] [--db_import] [--inspect] [--version_tb]
                   [--tag TAG] [--event_file PATH] [--path_prefix PATH]
                   [--window_title TEXT] [--max_reload_threads COUNT]
                   [--reload_interval SECONDS] [--reload_task TYPE]
                   [--reload_multifile BOOL]
                   [--reload_multifile_inactive_secs SECONDS]
                   [--generic_data TYPE]
                   [--samples_per_plugin SAMPLES_PER_PLUGIN]
                   [--detect_file_replacement BOOL]
                   {serve,dev} ...
tensorboard: error: invalid choice: '6006' (choose from 'serve', 'dev')
2024-04-18 16:26:12.148448: I tensorflow/tsl/cuda/cudart_stub.cc:28] Could not find cuda drivers on your machine, GPU will not be used.
2024-04-18 16:26:12.194512: I tensorflow/tsl/cuda/cudart_stub.cc:28] Could not find cuda drivers on your machine, GPU will not be used.
2024-04-18 16:26:12.195090: I tensorflow/core/platform/cpu_feature_guard.cc:182] This TensorFlow binary is optimized to use available CPU instructions in performance-critical operations.
To enable the following instructions: AVX2 FMA, in other operations, rebuild TensorFlow with the appropriate compiler flags.
2024-04-18 16:26:12.909553: W tensorflow/compiler/tf2tensorrt/utils/py_utils.cc:38] TF-TRT Warning: Could not find TensorRT
usage: tensorboard [-h] [--helpfull] [--logdir PATH] [--logdir_spec PATH_SPEC]
                   [--host ADDR] [--bind_all] [--port PORT]
                   [--reuse_port BOOL] [--load_fast {false,auto,true}]
                   [--extra_data_server_flags EXTRA_DATA_SERVER_FLAGS]
                   [--grpc_creds_type {local,ssl,ssl_dev}]
                   [--grpc_data_provider PORT] [--purge_orphaned_data BOOL]
                   [--db URI] [--db_import] [--inspect] [--version_tb]
                   [--tag TAG] [--event_file PATH] [--path_prefix PATH]
                   [--window_title TEXT] [--max_reload_threads COUNT]
                   [--reload_interval SECONDS] [--reload_task TYPE]
                   [--reload_multifile BOOL]
                   [--reload_multifile_inactive_secs SECONDS]
                   [--generic_data TYPE]
                   [--samples_per_plugin SAMPLES_PER_PLUGIN]
                   [--detect_file_replacement BOOL]
                   {serve,dev} ...
tensorboard: error: invalid choice: '6006' (choose from 'serve', 'dev')
2024-04-18 16:26:51.217165: I tensorflow/tsl/cuda/cudart_stub.cc:28] Could not find cuda drivers on your machine, GPU will not be used.
2024-04-18 16:26:51.264405: I tensorflow/tsl/cuda/cudart_stub.cc:28] Could not find cuda drivers on your machine, GPU will not be used.
2024-04-18 16:26:51.264965: I tensorflow/core/platform/cpu_feature_guard.cc:182] This TensorFlow binary is optimized to use available CPU instructions in performance-critical operations.
To enable the following instructions: AVX2 FMA, in other operations, rebuild TensorFlow with the appropriate compiler flags.
2024-04-18 16:26:52.046521: W tensorflow/compiler/tf2tensorrt/utils/py_utils.cc:38] TF-TRT Warning: Could not find TensorRT
usage: tensorboard [-h] [--helpfull] [--logdir PATH] [--logdir_spec PATH_SPEC]
                   [--host ADDR] [--bind_all] [--port PORT]
                   [--reuse_port BOOL] [--load_fast {false,auto,true}]
                   [--extra_data_server_flags EXTRA_DATA_SERVER_FLAGS]
                   [--grpc_creds_type {local,ssl,ssl_dev}]
                   [--grpc_data_provider PORT] [--purge_orphaned_data BOOL]
                   [--db URI] [--db_import] [--inspect] [--version_tb]
                   [--tag TAG] [--event_file PATH] [--path_prefix PATH]
                   [--window_title TEXT] [--max_reload_threads COUNT]
                   [--reload_interval SECONDS] [--reload_task TYPE]
                   [--reload_multifile BOOL]
                   [--reload_multifile_inactive_secs SECONDS]
                   [--generic_data TYPE]
                   [--samples_per_plugin SAMPLES_PER_PLUGIN]
                   [--detect_file_replacement BOOL]
                   {serve,dev} ...
tensorboard: error: invalid choice: '60012' (choose from 'serve', 'dev')
2024-04-18 16:28:00.307741: I tensorflow/tsl/cuda/cudart_stub.cc:28] Could not find cuda drivers on your machine, GPU will not be used.
2024-04-18 16:28:00.353348: I tensorflow/tsl/cuda/cudart_stub.cc:28] Could not find cuda drivers on your machine, GPU will not be used.
2024-04-18 16:28:00.353887: I tensorflow/core/platform/cpu_feature_guard.cc:182] This TensorFlow binary is optimized to use available CPU instructions in performance-critical operations.
To enable the following instructions: AVX2 FMA, in other operations, rebuild TensorFlow with the appropriate compiler flags.
2024-04-18 16:28:01.094340: W tensorflow/compiler/tf2tensorrt/utils/py_utils.cc:38] TF-TRT Warning: Could not find TensorRT
2024-04-18 16:28:03.463289: W tensorflow/core/common_runtime/gpu/gpu_device.cc:1960] Cannot dlopen some GPU libraries. Please make sure the missing libraries mentioned above are installed properly if you would like to use GPU. Follow the guide at https://www.tensorflow.org/install/gpu for how to download and setup the required libraries for your platform.
Skipping registering GPU devices...

NOTE: Using experimental fast data loading logic. To disable, pass
    "--load_fast=false" and report issues on GitHub. More details:
    https://github.com/tensorflow/tensorboard/issues/4784

TensorBoard 2.13.0 at http://cudahpc09:6012/ (Press CTRL+C to quit)
2024-04-18 16:30:02.346904: I tensorflow/tsl/cuda/cudart_stub.cc:28] Could not find cuda drivers on your machine, GPU will not be used.
2024-04-18 16:30:02.394268: I tensorflow/tsl/cuda/cudart_stub.cc:28] Could not find cuda drivers on your machine, GPU will not be used.
2024-04-18 16:30:02.394858: I tensorflow/core/platform/cpu_feature_guard.cc:182] This TensorFlow binary is optimized to use available CPU instructions in performance-critical operations.
To enable the following instructions: AVX2 FMA, in other operations, rebuild TensorFlow with the appropriate compiler flags.
2024-04-18 16:30:03.168888: W tensorflow/compiler/tf2tensorrt/utils/py_utils.cc:38] TF-TRT Warning: Could not find TensorRT
2024-04-18 16:30:04.323330: W tensorflow/core/common_runtime/gpu/gpu_device.cc:1960] Cannot dlopen some GPU libraries. Please make sure the missing libraries mentioned above are installed properly if you would like to use GPU. Follow the guide at https://www.tensorflow.org/install/gpu for how to download and setup the required libraries for your platform.
Skipping registering GPU devices...

NOTE: Using experimental fast data loading logic. To disable, pass
    "--load_fast=false" and report issues on GitHub. More details:
    https://github.com/tensorflow/tensorboard/issues/4784

TensorBoard 2.13.0 at http://cudahpc09:6013/ (Press CTRL+C to quit)
2024-04-18 16:31:35.612639: I tensorflow/tsl/cuda/cudart_stub.cc:28] Could not find cuda drivers on your machine, GPU will not be used.
2024-04-18 16:31:35.661982: I tensorflow/tsl/cuda/cudart_stub.cc:28] Could not find cuda drivers on your machine, GPU will not be used.
2024-04-18 16:31:35.662540: I tensorflow/core/platform/cpu_feature_guard.cc:182] This TensorFlow binary is optimized to use available CPU instructions in performance-critical operations.
To enable the following instructions: AVX2 FMA, in other operations, rebuild TensorFlow with the appropriate compiler flags.
2024-04-18 16:31:36.394804: W tensorflow/compiler/tf2tensorrt/utils/py_utils.cc:38] TF-TRT Warning: Could not find TensorRT
2024-04-18 16:31:37.435758: W tensorflow/core/common_runtime/gpu/gpu_device.cc:1960] Cannot dlopen some GPU libraries. Please make sure the missing libraries mentioned above are installed properly if you would like to use GPU. Follow the guide at https://www.tensorflow.org/install/gpu for how to download and setup the required libraries for your platform.
Skipping registering GPU devices...

NOTE: Using experimental fast data loading logic. To disable, pass
    "--load_fast=false" and report issues on GitHub. More details:
    https://github.com/tensorflow/tensorboard/issues/4784

TensorBoard 2.13.0 at http://cudahpc09:6014/ (Press CTRL+C to quit)
